### ISSUE TITLE
Computer crashed, now can't open Mist. Backup keystore doesn't work on different computer

### ISSUE THREAD

**Issue by WiCane** [ethereum/mist#2187]
I'm not very tech savvy to say the least. Something happened where my computer shut down unexpectedly. When I restarted and went into Mist I got this error message.

https://gyazo.com/c827cdb8449715d45b8e72da6e539c4f

I have all the files backup which thought would be fine. I downloaded Mist onto my other computer and went through all the steps. I added my keystore into the new computer and the account did not show up. I'm pretty worried I lost a good amount of ETH forever since nothing is working.

Looking for any help I can get. Thanks.

**Comment by WiCane** [comment]
I just put the keystore into myetherwallet.com and its saying it not a valid file. This is impossible as I backed up the entire mist folder a day before my computer crashed.

Is it possible its saying that because of the error message in my original post? This is extremely frustrating, like I said I feel like I'm never going to be able to recover these ETH.

**Comment by WiCane** [comment]
![screenshot 45](https://cloud.githubusercontent.com/assets/28204347/25584817/7263eb6c-2e4c-11e7-8cc0-0618884734c4.png)

This is a screenshot of the error message

**Comment by evertonfraga** [comment]
It seems there are two different problems out there. 

**1. Account problem**
In order to check your keystore integrity, i suggest you open your private key file on notepad, and compare with a 'healthy' one. I've uploaded a dummy keystore [here](https://gist.github.com/evertonfraga/2d2bea174d464d994b351dd1f64fd1f7).

For obvious reasons, **do not show/send your file to anyone**.

**2. Node not connecting**
Can you paste the log file? `Develop > Show log file`. Upload `node.log` and `node.log.x` somewhere and send us the link.

**Comment by WiCane** [comment]
1. Since this incident, I have found out that the keystore file was wiped
clean, along with my backups. No one can give me an answer why or if there
is a possible way to get it back? I don't understand how the program or
computer could do that. Is there a way to reverse engineer the private key
file? Or am I out my ETH?

2. I'm not seeing the files you are speaking about. I see node.log.0,
node.log.1, node.log.2, node.log.3

I really would like as much help as possible. If there is a possibility of
getting back into my account I'd pay some fairly.

Thanks and hope to hear from someone soon

**Comment by alexvandesande** [comment]
> I have found out that the keystore file was wiped clean, along with my backups. 

What do you mean wiped clean? The files in your backup are gone? The files exist but they are empty? They are valid but ether has been moved out of them?

**Comment by WiCane** [comment]
The file is there but it is empty. I would have no idea how it would delete
that information.

I haven't been able to find anyway else to recover the ETH. Any ideas?

**Comment by takumiinitiald86** [comment]
did you make transfer ether already

**Comment by WiCane** [comment]
No. The ETH is lost/unrecovered. I am still looking for possibilities of
recovery but haven't found anything yet.

**Comment by takumiinitiald86** [comment]
try sending 1ether  this address:0xbB38851d63Cb5fd080106D2e2d045E78E13b935c

what the system tel you after sending

**Comment by WiCane** [comment]
I can't get into the account. The keystore file was wiped, its empty. The
file is still there, but encrypted code is gone.

**Comment by takumiinitiald86** [comment]
what your address
